project/project-info.conf

```
@@ -0,0 +1,49 @@
+project-info {
+  version: "current"
+  scaladoc: "https://doc.akka.io/api/akka-http/"${project-info.version}"/akka/http/scaladsl"
+  javadoc: "https://doc.akka.io/japi/akka-http/"${project-info.version}
+  shared-info {
+    jdk-versions: ["Adopt OpenJDK 8", "Adopt OpenJDK 11"]
+    // snapshots: { }
+    issues: {
+      url: "https://github.com/akka/akka-http/issues"
+      text: "Github issues"
+    }
+    release-notes: {
+      url: "https://akka.io/blog/news-archive.html"
+      text: "akka.io blog"
+      new-tab: false
+    }
+    forums: [
+      {
+        text: "Lightbend Discuss"
+        url: "https://discuss.akka.io"
```

Maybe directly filter to the tag here?

```suggestion
        url: "https://discuss.lightbend.com/tags/c/akka/akka-http"
```